### ISSUE TITLE
Online API docs do not include types from SiliconStudio.Core.Mathematics

### ISSUE THREAD

**Issue by dfkeenan** [SiliconStudio/paradox#655]

**Comment by Kryptos-FR** [comment]
It is actually missing a lot of API from `Core`, not just `Mathematics`.

**Comment by dfkeenan** [comment]
Yeah, I noticed.  I am not sure if excluding `Core` things was on purpose or not.  I just thought `Mathematics` was kind of important. As it's where `Vector3`, `Matrix`, `Quaternion` etc. live.
